app/src/components/widgets/mostTimeConsumingTestCases/mostTimeConsumingTestCasesChart/getConfig.js

```diff
@@ -0,0 +1,104 @@
+import { PERIOD_VALUES } from 'common/constants/statusPeriodValues';
+import {
+  COLOR_TIMECONSUMING_BAR_RED,
+  COLOR_TIMECONSUMING_BAR_GREEN,
+} from 'common/constants/colors';
+import { CHART_MODES, MODES_VALUES } from 'common/constants/chartModes';
+import { MostTimeConsumingTooltip } from '../mostTimeConsumingTooltip/index';
+
+export const getConfig = ({
+  content,
+  intl,
+  positionCallback,
+  size: { height },
+  interval,
+  messages: MESSAGES,
+  chartType = MODES_VALUES[CHART_MODES.BAR_VIEW],
+  isPointsShow = true,
+  testCaseClickHandler,
+}) => {
+  const data = [...content.result]
+    .sort((a, b) => b.duration - a.duration)
+    .map((item, index) => ({ ...item, tickIndex: 30 - index, index }));
+
+  const getItemColor = (color, { index }) =>
+    index % 2 ? COLOR_TIMECONSUMING_BAR_GREEN : COLOR_TIMECONSUMING_BAR_RED;
```

Comment:
Color should be calculated based on item status, not index

Reply:
There are 2 stripe colors alternating each other - green and red, not depending on status